environments/backup-production/public.yml

```diff
@@ -0,0 +1,248 @@
+daily_deploy_email: tech-announce-daily@dimagi.com
+root_email: commcarehq-ops+root@dimagi.com
+server_email: commcarehq-noreply@dimagi.com
+server_admin_email: commcarehq-ops+admins@dimagi.com
+default_from_email: commcarehq-noreply-backup-production@dimagi.com
+return_path_email: commcarehq-bounces+backup-production@dimagi.com
+support_email: support@dimagi.com
+probono_support_email: pro-bono@dimagi.com
+accounts_email: accounts@dimagi.com
+data_email: datatree@dimagi.com
+subscription_change_email: accounts+subchange@dimagi.com
+internal_subscription_change_email: accounts+subchange+internal@dimagi.com
+billing_email: sales@dimagi.com
+invoicing_contact_email: billing-support@dimagi.com
+growth_email: growth@dimagi.com
+saas_ops_email: saas-ops@dimagi.com
+saas_reporting_email: saas-reporting@dimagi.com
+master_list_email: master-list@dimagi.com
+sales_email: sales@dimagi.com
+privacy_email: privacy@dimagi.com
+feedback_email: hq-feedback@dimagi.com
+eula_change_email: eula-notifications@dimagi.com
+contact_email: info@dimagi.com
+soft_assert_email: commcarehq-ops+soft_asserts@dimagi.com
+new_domain_email: inquiries@dimagi.com
+
+DATADOG_ENABLED: False
+
+DATADOG_TRACE:
+  TAGS:
+    env: backup-production
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_cluster_name: 'prodhqes-1.x'
+elasticsearch_version: 5.6.16
+elasticsearch_download_sha256: 6b035a59337d571ab70cea72cc55225c027ad142fbb07fd8984e54261657c77f.
+elasticsearch_memory: '4096m'
+#elasticsearch_node_name: '???'
+elasticsearch_fielddata_cache_size: "40%"
+elasticsearch_delete_old_logs: True
+
+nofile_limit: 65536
+
+backup_blobdb: False
+backup_postgres: plain
+backup_es_s3: False
+backup_couch: False
+postgres_s3: True
+postgresql_backup_days: 1
+postgresql_backup_weeks: 1
+nadir_hour: 16
+
+# pg_repack
+pg_repack_version: 1.4.7
+
+aws_region: 'us-east-2'
+
+filebeat_inputs:
+  - path: "{{ log_home }}/{{ deploy_env }}-timing.log"
+    tags: nginx-timing
+  - path: "{{ log_home }}/{{ deploy_env }}_commcare-nginx_error.log"
+    tags: nginx-error
+
+formplayer_java_version: "{{ java_17_bin_path }}/java"
+formplayer_archive_time_spec: '3d'
+formplayer_purge_time_spec: '10d'
+formplayer_sensitive_data_logging: true
+formplayer_forward_ip_proxy: true
+formplayer_detailed_tags:
+  - form_name
+  - module_name
+
+KSPLICE_ACTIVE: no # Don't know what this does, this is set to yes in prod
```

Comment:
This is fine. It's an old thing we don't really use anymore.

Reply:
Should I remove this?

Reply:
I think it's fine to leave. Do whatever staging has I guess